### PR TITLE
[UX] Remove K80 and M60 from common GPU list

### PR DESCRIPTION
**Pull request by Michaelvll**
Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `conda deactivate; bash -i tests/backward_compatibility_tests.sh`

### CODE REVIEW

File: sky/clouds/service_catalog/__init__.py [hunk 1]
```diff
@@ -324,13 +324,13 @@ def get_common_gpus() -> List[str]:
...
         'H100',
-        'K80',
         'L4',
-        'M60',
+        'L40S',
         'P100',
         'T4',
         'V100',
         'V100-32GB',
+        'MI25'
```
Comment (concretevitamin) [inline sky/clouds/service_catalog/__init__.py:333]: This isn't "common"; avoid?

File: sky/clouds/service_catalog/__init__.py [hunk 2]
```diff
@@ -340,10 +340,10 @@ def get_tpus() -> List[str]:
     # There are many TPU configurations available, we show the three smallest
     # and the largest configuration for the latest gen TPUs.
     return [
-        'tpu-v2-512', 'tpu-v3-2048', 'tpu-v4-8', 'tpu-v4-16', 'tpu-v4-32',
-        'tpu-v4-3968', 'tpu-v5litepod-1', 'tpu-v5litepod-4', 'tpu-v5litepod-8',
+        'tpu-v3-2048', 'tpu-v4-8', 'tpu-v4-16', 'tpu-v4-32',
```
Comment (concretevitamin) [inline sky/clouds/service_catalog/__init__.py:343]: 340-341 outdated now

File: sky/provision/kubernetes/utils.py [hunk 3]
```diff
@@ -1129,7 +1129,8 @@ def name(self) -> str:
         name = (f'{common_utils.format_float(self.cpus)}CPU--'
                 f'{common_utils.format_float(self.memory)}GB')
         if self.accelerator_count:
-            name += f'--{self.accelerator_count}{self.accelerator_type}'
+            acc_name = self.accelerator_type.replace(' ', '_')
```
Comment (concretevitamin) [inline sky/provision/kubernetes/utils.py:1132]: This is for...? Worth a comment.

File: sky/clouds/service_catalog/__init__.py (hunk 1 above)
Comment (Michaelvll) [reply sky/clouds/service_catalog/__init__.py:333]: This is the only AMD GPU that might be good to show.

File: sky/provision/kubernetes/utils.py (hunk 3 above)
Comment (Michaelvll) [reply sky/provision/kubernetes/utils.py:1135]: Updated

File: sky/clouds/service_catalog/__init__.py (hunk 2 above)
Comment (Michaelvll) [reply sky/clouds/service_catalog/__init__.py]: Updated

File: sky/clouds/service_catalog/__init__.py (hunk 1 above)
Comment (Michaelvll) [reply sky/clouds/service_catalog/__init__.py]: @concretevitamin wdyt?

Comment (concretevitamin) [reply sky/clouds/service_catalog/__init__.py]: I'd propose skipping.

**Review by concretevitamin — APPROVED** [review]

**Pull request merged**